pkg/registry/generic/options.go

```diff
@@ -16,13 +16,11 @@ limitations under the License.
 
 package generic
 
-import (
-	pkgstorage "k8s.io/kubernetes/pkg/storage"
-)
+import "k8s.io/kubernetes/pkg/storage/storagebackend"
 
 // RESTOptions is set of configuration options to generic registries.
 type RESTOptions struct {
-	Storage                 pkgstorage.Interface
+	Storage                 *storagebackend.Config
```

Comment:
Can you please rename Storage to StorageConfig (or sth that would actually mirror that it's not storage, but its config)?